var/spack/repos/builtin/packages/datatransferkit/package.py

```diff
@@ -11,13 +11,14 @@ class Datatransferkit(CMakePackage):
     parallel solution transfer services for multiphysics simulations"""
 
     homepage = "https://datatransferkit.readthedoc.io"
-    url      = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1-rc2.tar.gz"
+    url      = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1-rc3.tar.gz"
     git      = "https://github.com/ORNL-CEES/DataTransferKit.git"
 
     maintainers = ['Rombur']
 
     version('master', branch='master', submodules=True)
     version('3.1-rc2', commit='1abc1a43b33dffc7a16d7497b4185d09d865e36a', submodules=True)
+    version('3.1-rc3', commit='691d5a1540f7cd42141a3b3d2a7c8370cbc3560a', submodules=True)
```

Comment:
`rc3` is newer than `rc2` so it should be above `rc2`

Reply:
Fixed.